Protos/V1/events_common.proto

```diff
@@ -3,15 +3,31 @@ syntax = "proto3";
 package armonik.api.grpc.v1.events;
 
 import "result_status.proto";
+import "results_filters.proto";
 import "task_status.proto";
+import "tasks_filters.proto";
 
 option csharp_namespace = "ArmoniK.Api.gRPC.V1.Events";
 
+/**
+ * Represents the events that can be returned in the EventSubscriptionResponse
+ */
+enum EventsEnum {
+  EVENTS_ENUM_NEW_TASK = 0; /** New task */
```

Comment:
In Protobuf,iIt is recommended to have an "unspecified" filed that is invalid.

Reply:
added !